justin-tan/week5/tues/rails-crud-hw/db/seeds.rb

```diff
@@ -0,0 +1,55 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+#   Character.create(name: 'Luke', movie: movies.first)
+
+Restaurant.destroy_all 
+
+Restaurant.create!(
+    name: 'Carlton Wine Room',
+    chef: 'Conor Pomroy',
+    location: 'Carlton',
+    cuisine: 'Contemporary European',
+    year: 2018,
+    price_range: 50,
+    description: 'Elegant take on modern Australian food with a European influence that works well with wine and is seasonal.',
+    restaurant_img: 'app/assets/images/carlton-wine-room.jpg',
+    dish: 'Anchovy, fried bread, ricotta, pickled cucumber',
+    dish_img: 'app/assets/images/carlton-wine-room-dish.jpg'
```

Comment:
change all the relative paths to just the image name e.g. `'carlton-wine-room-dish.jpg'`

if you use the rails image tag helper, it will write the path for you (either local path or web URL)